hot-desk/src/main/resources/application.yml

```diff
@@ -5,8 +5,8 @@ datasource:
   schema: 'sandbox'
   host: '${dockerdbhost:localhost}:3306'
   url: 'jdbc:mariadb://${datasource.host}/${datasource.schema}'
-  username: 'root'
-  password: ''
+  username: 'mYsql'
+  password: 'mYsql934509018'
```

Comment:
Please do not add your credentials to our project

Reply:
Sorry, Done